### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rcookpalo**
Corrected two typos and made some grammatical corrections.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DeveloperTools/Integrations/CreateIncidents/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can execute these commands from the Cortex XSOAR CLI, as part of an automati
 After you successfully execute a command, a DBot message appears in the War Room with the command details.
 ### create-test-incident-from-file
 ***
-Creates incidents from json files provided, and stores it in the instance context.
+Creates incidents from json files provided, and stores them in the instance context.
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Integrations/CreateIncidents/README.md:23]:
```suggestion
Creates incidents from JSON files provided, and stores them in the instance context.
```

File: Packs/DeveloperTools/Integrations/CreateIncidents/README.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ Creates incidents from json files provided, and stores it in the instance contex
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| incidents_path | The path of json file containing incidents. Can contain one incident or a list of incidents. For example: Packs/somePack/TestPlaybooks/examples.json. | Required | 
-| attachment_paths | The paths of the files to be added to incidents as attachment. Would be added to all incidents provided in the incident_path file. For example: Packs/somePack/TestPlaybooks/attach.eml. | Optional | 
+| incidents_path | The path of the json file containing incidents. Can contain one incident or a list of incidents. For example: Packs/somePack/TestPlaybooks/examples.json. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Integrations/CreateIncidents/README.md:33]:
```suggestion
| incidents_path | The path of the JSON file containing incidents. Can contain one incident or a list of incidents. For example: Packs/somePack/TestPlaybooks/examples.json. | Required | 
```

File: Packs/DeveloperTools/Integrations/CreateIncidents/README.md [hunk 3]
```diff
@@ -56,8 +56,8 @@ Creates incidents from json file, and stores it in the instance context.
 | **Argument Name** | **Description** | **Required** |
 | --- | - |----------|
 | incident_raw_json | The json object that represents the incident. | Optional  | 
-| incident_entry_id | The enry ID of the json file that represents the incidents. | Optional  | 
-| attachment_paths | The paths of the files to be added to incidents as attachment. Would be added to all incidents provided in the incident_path file. For example: Packs/somePack/TestPlaybooks/attach.eml. | Optional | 
+| incident_entry_id | The entry ID of the json file that represents the incidents. | Optional  |
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Integrations/CreateIncidents/README.md:59]:
```suggestion
| incident_entry_id | The entry ID of the JSON file that represents the incidents. | Optional  | 
```

**Review by richardbluestone — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@rcookpalo Closing this PR and opening a new one under my name.  Because you are not a demisto member you can't contribute directly to master.  The new PR is https://github.com/demisto/content/pull/35879

**Pull request closed without merging**